src/dom/DomEvent.js

```diff
@@ -196,11 +196,7 @@ export function disableClickPropagation(el) {
 // with page reload when a `<form>` is submitted).
 // Use it inside listener functions.
 export function preventDefault(e) {
-	if (e.preventDefault) {
-		e.preventDefault();
-	} else {
-		e.returnValue = false;
-	}
+	e.preventDefault();

```

Comment:
At this point, perhaps we should consider getting rid of `DomEvent.preventDefault()` entirely, and call `preventDefault()` directly on the `Event`.